doc/dev/general.md

```diff
@@ -49,20 +49,34 @@ but only generic tools, i.e. `kubectl` and `helm`.
 the chosen foundation, and assuming that the cluster does not come
 with Helm Tiller pre-installed.
 
-### Deployment and Tear-down
+### Deployment and Tear-down - cf-operator
```

Comment:
Since this is a subsection of `cf-operator`, it doesn't make sense to append it again here. Same for the `kubecf` one.

Reply:
fixed